lmnet/executor/evaluate.py

```diff
@@ -25,6 +27,9 @@
 from lmnet.datasets.tfds import TFDSClassification, TFDSObjectDetection
 from lmnet.utils import config as config_util
 from lmnet.utils import executor, module_loader
+from lmnet.utils.predict_output.writer import save_json
+
+logging.basicConfig(level=logging.INFO)
```

Comment:
`logger = logging.getLogger(__name__)`

Reply:
and use `logger`.

Reply:
Thanks! @kchygoe 
I changed it to use logger and I also fixed `lmnet.utils.predict_output.writer` as same.
The log is changed to show module name (instead of `root`) as following.
```
INFO:__main__:test_step_size3
INFO:__main__:test_step0
INFO:__main__:test_step1
INFO:__main__:test_step2
INFO:lmnet.utils.predict_output.writer:save json: /home/blueoil/saved/train_20191216084759/evaluate/json/27.json
```